node/core/provisioner/src/lib.rs

```diff
@@ -235,7 +235,11 @@ impl ProvisioningJob {
 		)
 		.await
 		{
-			tracing::warn!(target: LOG_TARGET, err = ?err, "failed to assemble or send inherent data");
+			// THIS IS A HACK: to silence warnings on live chains without the parachains runtime
```

Comment:
Even on a live chain, this should not print a warning, because the receiver side was dropped.